<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-04-07T15:22:39Z" name="Command line suite" finished-at="2020-04-07T15:23:09Z" duration-ms="30061">
    <groups>
    </groups>
    <test started-at="2020-04-07T15:22:39Z" name="Command line test" finished-at="2020-04-07T15:23:09Z" duration-ms="30061">
      <class name="com.Jenkins.BasicMavenSeleniumTest.SeleniumTest2">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Jenkins.BasicMavenSeleniumTest.SeleniumTest2@7ce3cb8e]" started-at="2020-04-07T15:22:39Z" name="setup" finished-at="2020-04-07T15:22:44Z" duration-ms="5606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test2()[pri:0, instance:com.Jenkins.BasicMavenSeleniumTest.SeleniumTest2@7ce3cb8e]" started-at="2020-04-07T15:22:44Z" name="test2" finished-at="2020-04-07T15:22:50Z" duration-ms="5388" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="after()[pri:0, instance:com.Jenkins.BasicMavenSeleniumTest.SeleniumTest2@7ce3cb8e]" started-at="2020-04-07T15:22:50Z" name="after" finished-at="2020-04-07T15:22:54Z" duration-ms="4188" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- com.Jenkins.BasicMavenSeleniumTest.SeleniumTest2 -->
      <class name="com.Jenkins.BasicMavenSeleniumTest.SeleniumTest3">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Jenkins.BasicMavenSeleniumTest.SeleniumTest3@78b66d36]" started-at="2020-04-07T15:22:54Z" name="setup" finished-at="2020-04-07T15:22:59Z" duration-ms="5129" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test3()[pri:0, instance:com.Jenkins.BasicMavenSeleniumTest.SeleniumTest3@78b66d36]" started-at="2020-04-07T15:22:59Z" name="test3" finished-at="2020-04-07T15:23:04Z" duration-ms="5358" status="FAIL">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<...1> but was:<...>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<...1> but was:<...>
at junit.framework.Assert.assertEquals(Assert.java:81)
at junit.framework.Assert.assertEquals(Assert.java:87)
at com.Jenkins.BasicMavenSeleniumTest.SeleniumTest3.test3(SeleniumTest3.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method is-config="true" signature="after()[pri:0, instance:com.Jenkins.BasicMavenSeleniumTest.SeleniumTest3@78b66d36]" started-at="2020-04-07T15:23:04Z" name="after" finished-at="2020-04-07T15:23:09Z" duration-ms="4363" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- com.Jenkins.BasicMavenSeleniumTest.SeleniumTest3 -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
